cypress/e2e/marc/marc-authority/verify-authority-source-facet-option.cy.js

```diff
@@ -0,0 +1,87 @@
+import Permissions from '../../../support/dictionary/permissions';
+import DataImport from '../../../support/fragments/data_import/dataImport';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import MarcAuthorities from '../../../support/fragments/marcAuthority/marcAuthorities';
+import MarcAuthority from '../../../support/fragments/marcAuthority/marcAuthority';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import { JOB_STATUS_NAMES } from '../../../support/constants';
+
+describe('MARC -> MARC Authority', () => {
+  const testData = {
+    searchOption: 'Personal name',
+    searchValue: 'France',
+    authorityOption: 'LC Subject Headings (LCSH)',
+  };
+
+  const marcFiles = [
+    {
+      marc: 'marcAuthFileC365626.mrc',
+      fileName: `testMarcFileC365626.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      numOfRecords: 1,
+    },
+  ];
+
+  const createdRecordIDs = [];
+
+  before('Create test data', () => {
+    cy.loginAsAdmin({
+      path: TopMenu.dataImportPath,
+      waiter: DataImport.waitLoading,
+    }).then(() => {
+      marcFiles.forEach((marcFile) => {
+        cy.visit(TopMenu.dataImportPath);
+        DataImport.verifyUploadState();
+        DataImport.uploadFileAndRetry(marcFile.marc, marcFile.fileName);
+        JobProfiles.waitFileIsUploaded();
+        JobProfiles.waitLoadingList();
+        JobProfiles.search(marcFile.jobProfileToRun);
+        JobProfiles.runImportFile();
+        JobProfiles.waitFileIsImported(marcFile.fileName);
+        Logs.checkStatusOfJobProfile(JOB_STATUS_NAMES.COMPLETED);
+        Logs.openFileDetails(marcFile.fileName);
+        for (let i = 0; i < marcFile.numOfRecords; i++) {
+          Logs.getCreatedItemsID(i).then((link) => {
+            createdRecordIDs.push(link.split('/')[5]);
+          });
+        }
+      });
+    });
+
+    cy.createTempUser([Permissions.uiMarcAuthoritiesAuthorityRecordView.gui]).then(
+      (createdUserProperties) => {
+        testData.userProperties = createdUserProperties;
+        cy.login(testData.userProperties.username, testData.userProperties.password, {
+          path: TopMenu.marcAuthorities,
+          waiter: MarcAuthorities.waitLoading,
+        });
+      },
+    );
+  });
+
+  after('Delete test data', () => {
+    cy.getAdminToken();
+    Users.deleteViaApi(testData.userProperties.userId);
+    MarcAuthority.deleteViaAPI(createdRecordIDs[0]);
+  });
+
+  it(
+    'C365626 Search | Verify that the "Authority source" facet option will display the name of facet option when zero results are returned (Spitfire) (TaaS)',
```

Comment:
Check for zero counter value in authority source facet should be added according to the last step of the test case
BUT - we have a bug in the application at this moment - [UIMARCAUTH-363](https://issues.folio.org/browse/UIMARCAUTH-363)
Please postpone automation of this test case. I will mark it as "Review by MQA" until issue is resolved

Reply:
the bug was fixed & closed